<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeManagerWidgetClass</class>
 <widget class="QWidget" name="EmployeeManagerWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>642</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EmployeeManagerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_main">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lb_p1_overview">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Employee Overview</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="2" column="1">
                 <widget class="QLabel" name="lb_p1_location">
                  <property name="text">
                   <string>Company Location</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lb_p1_firstName">
                  <property name="text">
                   <string>First Name</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="cb_p1_companyLoc"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="le_p1_firstName"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_p1_lastName">
                  <property name="text">
                   <string>Last Name</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="le_p1_lastName"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="lb_p1_phone">
                  <property name="text">
                   <string>Phone</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="le_p1_phone"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="chb_p1_deletedEmployee">
                <property name="text">
                 <string>Include deleted Employee</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_p1_reset">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Actions</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="1" column="0">
                 <widget class="QPushButton" name="pb_p1_delete">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pb_p1_edit">
                  <property name="text">
                   <string>Edit</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QPushButton" name="pb_p1_add">
                  <property name="text">
                   <string>Add new</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tv_employee"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_addEdit">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Data</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="2" column="0">
                 <widget class="QLabel" name="lb_p2_phone">
                  <property name="text">
                   <string>Phone</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="le_p2_firstName"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="le_p2_lastName"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_p2_lastName">
                  <property name="text">
                   <string>Last Name</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lb_p2_firstName">
                  <property name="text">
                   <string>First Name</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="le_p2_phone"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="lb_p2_location">
                  <property name="text">
                   <string>Company Location</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="cb_p2_companyLoc"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Actions</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="1" column="0">
                 <widget class="QPushButton" name="pb_p2_clear">
                  <property name="text">
                   <string>Clear</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pb_p2_back">
                  <property name="text">
                   <string>Back</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QPushButton" name="pb_p2_save">
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_p2_firstName</tabstop>
  <tabstop>le_p2_lastName</tabstop>
  <tabstop>le_p2_phone</tabstop>
  <tabstop>cb_p2_companyLoc</tabstop>
  <tabstop>pb_p2_save</tabstop>
  <tabstop>pb_p2_clear</tabstop>
  <tabstop>pb_p2_back</tabstop>
  <tabstop>cb_p1_companyLoc</tabstop>
  <tabstop>pb_p1_delete</tabstop>
  <tabstop>tv_employee</tabstop>
  <tabstop>le_p1_firstName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
